controllers/chat/group/changeChannelDetail.js

```diff
@@ -38,38 +41,63 @@ const changeChannelDetail = async (req, res) => {
       const textTargetUser = `${ownUser.username} changed the group name to "${channel_name}"`;
       const textDefaultUser = `${ownUser.username} changed the group name to "${channel_name}"`;
 
-      await channel.sendMessage({
-        text: textDefaultUser,
-        own_text: textOwnUser,
-        other_text: textTargetUser,
-        better_type: 'change_channel_detail',
-        type: 'system',
-        user_id: userId,
-        only_to_user_show: userId,
-        disable_to_user: false,
-        is_from_prepopulated: true,
-        system_user: userId,
-        isSystem: true,
-        members: members
-      });
+      await channel.sendMessage(
+        {
+          text: textDefaultUser,
+          own_text: textOwnUser,
+          other_text: textTargetUser,
+          better_type: 'change_channel_detail',
+          type: 'system',
+          user_id: userId,
+          only_to_user_show: userId,
+          disable_to_user: false,
+          is_from_prepopulated: true,
+          system_user: userId,
+          isSystem: true,
+          members: members
+        },
+        {
+          skip_push: true
+        }
+      );
+
+      const other_members = all_members.filter((m) => m !== userId);
+      await Promise.all(
+        other_members.map(async (m) => {
+          await BetterSocialCore.fcmToken.sendGroupChatNotification(m, textDefaultUser);
+        })
+      );
     }
 
     if (channel_image) {
       const textOwnUser = `You changed the group image`;
       const textTargetUser = `${ownUser.username} changed the group image`;
-      const textOtherUser = `${ownUser.username} changed the group image`;
-
-      await channel.sendMessage({
-        text: textOtherUser,
-        own_text: textOwnUser,
-        other_text: textTargetUser,
-        better_type: 'change_channel_detail',
-        type: 'system',
-        user_id: userId,
-        system_user: userId,
-        isSystem: true,
-        members: members
-      });
+      const textDefaultUser = `${ownUser.username} changed the group image`;
+
+      await channel.sendMessage(
+        {
+          text: textDefaultUser,
+          own_text: textOwnUser,
+          other_text: textTargetUser,
+          better_type: 'change_channel_detail',
+          type: 'system',
+          user_id: userId,
+          system_user: userId,
+          isSystem: true,
+          members: members
+        },
+        {
+          skip_push: true
+        }
+      );
+
+      const other_members = all_members.filter((m) => m !== userId);
+      await BetterSocialCore.fcmToken.sendGroupChatNotification(userId, textOwnUser);
+      await Promise.all(
+        other_members.map(async (m) => {
+          await BetterSocialCore.fcmToken.sendGroupChatNotification(m, textDefaultUser);
+        })
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure that `channel.sendMessage` supports the additional options and that `BetterSocialCore.fcmToken.sendGroupChatNotification` handles errors properly. Also, it seems like a notification is being sent to the user who made the change (`userId`). If this is intentional, make sure it's necessary as it could lead to unnecessary notifications.
```diff
+ await channel.sendMessage(
+   {
+     text: textDefaultUser,
+     own_text: textOwnUser,
+     other_text: textTargetUser,
+     better_type: 'change_channel_detail',
+     type: 'system',
+     user_id: userId,
+     system_user: userId,
+     isSystem: true,
+     members: members
+   },
+   {
+     skip_push: true
+   }
+ );
+ 
+ const other_members = all_members.filter((m) => m !== userId);
+ await BetterSocialCore.fcmToken.sendGroupChatNotification(userId, textOwnUser);
+ await Promise.all(
+   other_members.map(async (m) => {
+     await BetterSocialCore.fcmToken.sendGroupChatNotification(m, textDefaultUser);
+   })
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->